### ISSUE TITLE
Create Brochures and Promo Materials for FA 2017 offerings

### ISSUE THREAD

**Issue by zee-em** [zee-em/saw-program#7]
GOAL: have some really nice print materials to hand out at SPX and other events, plus to distribute locally as needed. Digital versions to be shared via social media, etc. 

- [ ] gather content and copy 
- [ ] create guidelines for look & feel / style guide
- [ ] generate graphics
- [ ] devise process for review/sign-offs
...

**Comment by zee-em** [comment]
@hutchowen have you talked to lisa lately about graphic design?

**Comment by hutchowen** [comment]
Yes we're talking about using some Miranda Harmon art. I am retrieving this from her, and Lisa and I will be meeting soon. Plan is to get brochure out by Sept
